### PR TITLE
Added dialogues for privacy info and each of the modes

### PR DESCRIPTION
**Pull request by alveerak**

### CODE REVIEW

**Comment by alveerak** [comment]
Also fixed async Parse user login and Homescreen Activity launch. Since the user wasn't logged in sometimes (aka null) in time for the home screen activity launch, the entire home screen would sometimes show up without the profile pic and user info.

**Comment by alveerak** [comment]
Also made a minor layout change for home screen landscape mode.

File: app/src/main/java/a3/com/convo/activities/LoginActivity.java [hunk 1]
```diff
@@ -149,8 +149,9 @@ public void onSuccess(LoginResult loginResult) {
                         return;
                     }
                     getUserInfo(at);
-                    Intent i = new Intent(LoginActivity.this, HomeScreenActivity.class);
-                    startActivity(i);
+//                    while (ParseUser.getCurrentUser() == null) {
```
Comment (samhithak) [inline app/src/main/java/a3/com/convo/activities/LoginActivity.java]: remove this?

File: app/src/main/res/layout/fragment_mode.xml [hunk 2]
```diff
@@ -76,4 +77,45 @@
...
     </LinearLayout>
+
+    <ImageView
+        android:id="@+id/freestyle_info"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignParentEnd="true"
+        android:layout_alignParentTop="true"
+        android:layout_marginEnd="54dp"
+        android:layout_marginTop="158dp"
```
Comment (samhithak) [inline app/src/main/res/layout/fragment_mode.xml]: move this to dimens later

**Pull request merged**